packages/components/molecules/f-rating/src/tests/constants.test.js

```diff
@@ -26,4 +27,17 @@ describe('`constants`', () => {
             expect(result).toBe(true);
         });
     });
+
+    describe('`VALID_STAR_FONT_SIZES`', () => {
```

Comment:
What's the purpose or value of this test? Isn't this testing the definition of the constant itself?

Reply:
Could remove but usually I prefer to make sure the constant file is protected from inadvertent changes, this test will make sure the constant file contains the correct  expected values at all times.

Reply:
ok, tbh I've never tested a constant like this - if you were testing a validator then it would make sense to test the validator returns true if it matches the constant and false otherwise, but in this case, you could set in the test like the following and it would pass too:

```
const values = ['default', 'large', 'foo', 'bar'];
```

No strong feelings though, so will approve :D

Reply:
No that raises a good point. I'll take a look at this tomorrow. 👍

Reply:
@juan-amor Fixed now.